src/components/charts/BarChart.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="bar-chart-container" v-if="loaded">
+    <Bar
+      id="my-chart-id"
+      :options="chartOptions"
+      :data="chartData"
+    />
+  </div>
+  <div v-else>
+    Loading...
+  </div>
+</template>
+
+<script>
+import { Bar } from 'vue-chartjs';
+import { Chart as ChartJS, Title, Tooltip, Legend, BarElement, CategoryScale, LinearScale } from 'chart.js';
+
+ChartJS.register(Title, Tooltip, Legend, BarElement, CategoryScale, LinearScale);
+
+export default {
+  name: 'BarChart',
+  components: { Bar },
+  data() {
+    return {
+      loaded: false,
+      chartOptions: {
+        animation: false,
+        transitions: {
+          active: {
+            animation: {
+              duration: 0
+            }
+          }
+        },
+        responsive: true,
+        maintainAspectRatio: false,
+        scales: {
+          x: {
+            stacked: true
+          },
+          y: {
+            stacked: true
+          }
+        },
+        plugins: {
+          title: {
+            display: true,
+            text: "Croom" // Default title
+          }
+        }
+      },
+      chartData: {
+        labels: ['Free', 'Occupied'],
+        datasets: [
+          {
+            label: 'Bed Status',
+            data: [],  
+            backgroundColor: ['rgba(75, 192, 192, 0.2)', 'rgba(255, 99, 132, 0.2)'],
+            borderColor: ['rgba(75, 192, 192, 1)', 'rgba(255, 99, 132, 1)'],
+            borderWidth: 1
+          }
+        ]
+      },
+    };
+  },
+ 
```

Comment:
Add a prop section here with called hospital which is required and takes an object eg
```js
    props: {
        hospital: {
            type: Object,
            required: true
        },
   }
```